npmosf/include/node/android-ifaddrs.h

```diff
@@ -0,0 +1,54 @@
+/*
```

Comment:
Am I correct in understanding that these files are packages installed by npm? Is this the default location or have you added these manually? In general we try not to keep node modules in the git index. 

If there's not a special reason to keep these files, can you please do a `git rm -r --cached npmosf` to remove these file from your index? Should make this PR look a lot smaller :).

You will also need to move these files or update the .gitignore to keep them from being tracked again. The existing .gitignore will not index the node_modules directory (npm's default install location), so unless you have a special npm configuration `mv npmosf node_modules` should work fine. Otherwise updating the .gitignore entry with the correct path should work as well.